Source/Scene/Cesium3DTileBatchTable.js

```diff
@@ -0,0 +1,1519 @@
+                        // componentCount (e.g. a VEC4 would be 4), and the type used to pack and unpack (e.g. Cartesian4).
+                        binaryProperties[name] = {
+                            typedArray : typedArray,
+                            componentCount : componentCount,
+                            type : classType
+                        };
+                    }
+                }
+            }
+        }
+        return binaryProperties;
+    };
+
+    function getByteLength(batchTable) {
+        var dimensions = batchTable._textureDimensions;
+        return (dimensions.x * dimensions.y) * 4;
+    }
+
+    function getBatchValues(batchTable) {
+        if (!defined(batchTable._batchValues)) {
+            // Default batch texture to RGBA = 255: white highlight (RGB) and show/alpha = true/255 (A).
+            var byteLength = getByteLength(batchTable);
+            var bytes = new Uint8Array(byteLength);
+            arrayFill(bytes, 255);
+            batchTable._batchValues = bytes;
+        }
+
+        return batchTable._batchValues;
+    }
+
+    function getShowAlphaProperties(batchTable) {
+        if (!defined(batchTable._showAlphaProperties)) {
+            var byteLength = 2 * batchTable.featuresLength;
+            var bytes = new Uint8Array(byteLength);
+            // [Show = true, Alpha = 255]
+            arrayFill(bytes, 255);
+            batchTable._showAlphaProperties = bytes;
+        }
+        return batchTable._showAlphaProperties;
+    }
+
+    Cesium3DTileBatchTable.prototype.setShow = function(batchId, show) {
+        var featuresLength = this.featuresLength;
+        //>>includeStart('debug', pragmas.debug);
+        if (!defined(batchId) || (batchId < 0) || (batchId > featuresLength)) {
+            throw new DeveloperError('batchId is required and between zero and featuresLength - 1 (' + featuresLength - + ').');
+        }
+
+        if (!defined(show)) {
+            throw new DeveloperError('show is required.');
+        }
+        //>>includeEnd('debug');
+
+        if (show && !defined(this._showAlphaProperties)) {
+            // Avoid allocating since the default is show = true
+            return;
+        }
+
+        var showAlphaProperties = getShowAlphaProperties(this);
+        var propertyOffset = batchId * 2;
+
+        var newShow = show ? 255 : 0;
+        if (showAlphaProperties[propertyOffset] !== newShow) {
+            showAlphaProperties[propertyOffset] = newShow;
+
+            var batchValues = getBatchValues(this);
+
+            // Compute alpha used in the shader based on show and color.alpha properties
+            var offset = (batchId * 4) + 3;
+            batchValues[offset] = show ? showAlphaProperties[propertyOffset + 1] : 0;
+
+            this._batchValuesDirty = true;
+        }
+    };
+
+    Cesium3DTileBatchTable.prototype.setAllShow = function(show) {
```

Comment:
Even though its not used anywhere else its nice to have as a mirror of `setAllColor`.